Main.cpp

```diff
@@ -14,6 +14,7 @@ int main()
 
 	int tNumber = 0;
```

Comment:
As I suggested earlier, make the variable name more descriptive so anyone reading your code will get a better idea.